assets/js/modules/adsense/components/dashboard/DashboardSummaryWidget.js

```diff
@@ -77,12 +99,15 @@ function DashboardSummaryWidget( { Widget, WidgetReportZero, WidgetReportError }
 		return {
 			today: select( STORE_NAME ).getReport( todayArgs ),
```

Comment:
With these changes, we should have no more need for the today-specific report and related logic so I think this (and related) should be removed.